extensions/persistence/org.eclipse.smarthome.persistence.h2sql/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="java"/>
+	<classpathentry kind="lib" path="lib/h2-1.3.176.jar"/>
```

Comment:
There are CQs for other Eclipse projects that are using h2.
The most recent CQ is for "1.4.186".
If we need to use a 1.3 one, the most recent here is: 1.3.169

I assume the CQ will be applied more recently, if we use a version another approved CQ already exist.